### PR TITLE
Add API snippet zero-shot img classification

### PR DESCRIPTION
**Pull request by osanseviero**
TODO (in other PRs)

- [x] Zero shot img classification
- [ ] Image to image
- [ ] Tabular Classification
- [ ] Tabular Regression
- [ ] Visual Question Answering
- [ ] Document Question Answering
- [ ] Zero shot object detection ? (need to investigate this one a bit more)

### CODE REVIEW

File: packages/tasks/src/snippets/python.ts [hunk 1]
```diff
@@ -12,6 +12,22 @@ output = query({
...
+		img = f.read()
+	payload={
+		"parameters": data["parameters"],
+		"inputs": base64.b64encode(img).decode("utf-8")
+	}
+	response = requests.post(API_URL, headers=headers, json=payload)
+	return response.json()
+
+output = query({
+    "inputs": ${getModelInputSnippet(model)},
```
Comment (merveenoyan) [inline packages/tasks/src/snippets/python.ts:27]: I'm a little confused, aren't we passing the example input (which is the image path in the other file)?
if so, isn't there supposed to be an `image_path` key in the dict that is passed to the query function? 

if so it should be:
output = query({
    "image_path": ${getModelInputSnippet(model)},

**Review by merveenoyan — COMMENTED** [review]
thank you!

File: packages/tasks/src/snippets/python.ts (hunk 1 above)
Comment (osanseviero) [reply packages/tasks/src/snippets/python.ts]: Good catch! Fixed it

File: packages/tasks/src/snippets/python.ts [hunk 2]
```diff
@@ -12,6 +12,22 @@ output = query({
     "parameters": {"candidate_labels": ["refund", "legal", "faq"]},
 })`;
 
+export const snippetZeroShotImageClassification = (model: ModelData): string =>
+	`def query(data):
+	with open(data["image_path"], "rb") as f:
+		img = f.read()
+	payload={
+		"parameters": data["parameters"],
+		"inputs": base64.b64encode(img).decode("utf-8")
```
Comment (pcuenca) [inline packages/tasks/src/snippets/python.ts:21]: Interesting! So the input image is received as binary data, I guess. Is there a way to test and debug these snippets before merging?

**Review by pcuenca — APPROVED** [review]
Looks good!

File: packages/tasks/src/snippets/python.ts (hunk 2 above)
Comment (osanseviero) [reply packages/tasks/src/snippets/python.ts:21]: Yes, you can test the python snippet locally or in Colab

```python
import requests
import base64

API_URL = "https://api-inference.huggingface.co/models/openai/clip-vit-base-patch32"

def query(data):
	with open(data["image_path"], "rb") as f:
		img = f.read()
	payload={
		"parameters": data["parameters"],
		"image":  base64.b64encode(img).decode("utf-8"),
	}
	response = requests.post(API_URL, json=payload)
	return response.json()
 
output = query({
	"image_path": "/content/download.png",
	"parameters": {"candidate_labels": ["cat", "dog", "llama"]},
})
```

for example

Comment (pcuenca) [reply packages/tasks/src/snippets/python.ts:21]: Oh, of course! Thanks a lot! 🙌

**Pull request merged**